app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -96,4 +119,7 @@ export default class MoreConfig extends Component {
 
 MoreConfig.propTypes = {
   onExit: PropTypes.func.isRequired,
+  passedData: PropTypes.objectOf().isRequired,
```

Comment:
You can use PropTypes.shape({...}) to define this based on the shape at this line 👇🏻 https://github.com/thepracticaldev/dev.to/blob/f14efbd65a71e4683368138ca6ba3b44879b5d8c/app/javascript/article-form/elements/moreConfig.jsx#L32